evacuees/src/API/EMBC.Registrants.API/Utils/EmailSender.cs

```diff
@@ -145,14 +147,14 @@ public void Send(EmailMessage emailMessage)
 
                 try
                 {
-                    if (emailConfiguration.SmtpServer == BcGovSmtpServer)
+                    if (emailConfiguration.EnableSSL == false)
```

Comment:
change to a single trinary if

Reply:
I couldn't find a way to do this in a way that worked and made sense to me. Perhaps you can show me what you had in mind.

Reply:
Sure, let me know when you're available for live share